pdc/apps/package/models.py

```diff
@@ -191,48 +191,9 @@ def clean(self):
     @property
     def parsed_version(self):
         if not hasattr(self, '_version'):
-            self._version = parse_epoch_version(self.version)
+            self._version = parse_epoch_version(self.version) if self.version else None
```

Comment:
I guess this change fix the bug

Reply:
Yes, this allows the dependency to be just "python" instead of "python>=2.7".